src/integrationTest/java/uk/gov/hmcts/reform/wataskmanagementapi/services/MIReportingServiceTest.java

```diff
@@ -93,12 +101,46 @@ void should_save_task_and_get_task_from_replica_tables() {
                     assertEquals(1, taskHistoryResourceList.size());
                     assertEquals(taskResource.getTaskId(), taskHistoryResourceList.get(0).getTaskId());
                     assertEquals(taskResource.getTaskName(), taskHistoryResourceList.get(0).getTaskName());
+                    assertEquals(taskResource.getTitle(), taskHistoryResourceList.get(0).getTaskTitle());
 
                     return true;
                 });
     }
 
+    @Test
+    void should_save_task_and_get_task_from_reportable_task() {
+        TaskResource taskResource = createAndSaveTask();
+
+        await().ignoreException(AssertionFailedError.class)
+            .pollInterval(1, SECONDS)
+            .atMost(10, SECONDS)
+            .until(
+                () -> {
+                    List<ReportableTaskResource> reportableTaskList
+                        = miReportingService.findByReportingTaskId(taskResource.getTaskId());
+
+                    assertFalse(reportableTaskList.isEmpty());
+                    assertEquals(1, reportableTaskList.size());
+                    assertEquals(taskResource.getTaskId(), reportableTaskList.get(0).getTaskId());
+                    assertEquals(taskResource.getTaskName(), reportableTaskList.get(0).getTaskName());
+                    assertEquals(taskResource.getTitle(), reportableTaskList.get(0).getTaskTitle());
```

Comment:
We could extend the assertions to guarantee the taskHistory Table will not have

1.  a task with earlier timestamp
2. Assert task attributes, last_updated, task_state & last_reconfiguration_time to ensure the eventual state is consistent iin our reportable task table

Reply:
Added checks for attributes on reportable task table.